README.md

```diff
@@ -1,114 +1,80 @@
-solutions, and minimum rank corresponds to most preferred, these naturally 
-align.
-
-### Even Distribution
-
-An even distribution of employees across shifts would have approximately
-`num_employees/num_shifts` scheduled employees per shift. To enforce this
-requirement, both linear and quadratic biases must be adjusted in a specific
-manner. We can do this either by building an equality constraint in our 
-constrained quadratic model, or using linear and quadratic biases to enforce 
-the constraint in the discrete quadratic model.
-
-To determine the linear and quadratic bias adjustments, we must consider the
-underlying binary variables in our model. For a DQM with N shifts and M employees,
-each employee has a single variable constructed with N cases or classes. These
-are implemented as N binary variables per employee &mdash; one for each possible
-shift.
-
-For a specific shift `i`, we require that exactly `M/N` employees are scheduled.
-This is equivalent to saying that `M/N` employee variables are assigned case
-`i`, or, returning to our binary variables, that `M/N` of the binary variables
-corresponding to case `i` take a value of 1. In other words, the sum of *all*
-employee case `i` binary variables should equal `M/N`. An equality over a
-summation of binary variables can be converted to a minimization expression by
-moving from the equality:
-
-```sum(shift i binary variables) = M/N```
-
-to a minimization expression:
-
-```min( ( sum(shift i binary variables) - M/N)**2 )```
-
-Expanding and simplifying this expression of binary variables, it becomes:
-
-```min( (-2*M/N+1)*sum(shift i linear biases) + 2*sum(shift i quadratic biases))```
-
-When this constraint is built into our DQM object, it is added in with a
-coefficient `gamma`.  This term gamma is known as a Lagrange parameter and can
-be used to weight this constraint against the competing employee preferences.
-You may wish to adjust this parameter depending on your problem requirements and
-size. The value set here in this program was chosen to empirically work well as
-a starting point for problems of a wide-variety of sizes. For more information
-on setting this parameter, see D-Wave's [Problem Formulation Guide](https://www.dwavesys.com/practical-quantum-computing-developers).
+- Allow isolated days off: Unchecked, this option means that no employee can
+  have a stand-alone day off. Every day off must be at least two consecutive
+  days.
+- Require a manager on every shift: Checked, this option means that every shift
+  must have exactly one manager on duty to supervise.
+- Min/max shifts per employee: The range that determines the number of shifts an
+  employee can work in the month.
+- Min/max employees per shift: The range that determines how many employees need
+  to be assigned to each shift.
+- Max consecutive shifts: The maximum number of shifts in a row that an employee
+  can be scheduled before they need to have a day off.
```

Comment:
```suggestion
```

Should be moved to under ###Inputs per the updated [example template](https://github.com/dwave-examples/template)

I edited the phrasing in a separate suggested change under ###Inputs